src/wallet/Wallet.js

```diff
@@ -19,15 +20,17 @@ const Wallet = props => (
       </Affix>
       <Affix className="rightContainer" stickPosition={77}>
         <div className="right">
-          <SteemTrendingCharts />
           <Action
-            style={{ margin: '10px 0' }}
+            style={{ marginBottom: '10px' }}
             text={props.intl.formatMessage({
               id: 'transfer',
               defaultMessage: 'Transfer',
             })}
             onClick={() => props.openTransfer('')}
```

Comment:
I think that we can use handler and move `primary` up here as well, as it's not a big change and somewhat related to this PR.